s3same/iam.py

```diff
@@ -29,16 +29,20 @@ def _policy_string(bucket):
             ],
         })
 
-def _find_policy(iam):
-    kwargs = {'Scope': 'Local'}
+def _all_pages(iam_method, item_key, **kwargs):
     while True:
-        response = iam.list_policies(**kwargs)
-        for policy in response.get('Policies', []):
-            if policy.get('PolicyName') == IAMName:
-                return policy
+        response = iam_method(**kwargs)
+        for key in response.get(item_key, []):
+            yield key
         kwargs['Marker'] = response.get('Marker')
         if not response.get('IsTruncated') or kwargs['Marker'] is None:
-            return None
+            return
+
+def _find_policy(iam):
+    for policy in _all_pages(iam.list_policies, 'Policies', Scope='Local'):
+        if policy.get('PolicyName') == IAMName:
+            return policy
+    return None
```

Comment:
Heh, see [this](https://github.com/vokal/s3same/pull/1#discussion-diff-54206831).